enums/src/macros.rs

```diff
@@ -23,10 +14,15 @@ macro_rules! mk_enum {
 macro_rules! mk_get_language {
     ( $( ($camel:ident, $name:ident) ),* ) => {
         pub fn get_language(lang: &LANG) -> Language {
-            unsafe {
+              if let LANG::Java = lang {
```

Comment:
It's a pain that we have to specify all languages manually now :(

Reply:
The other solution would be splitting up the big macro in `enums/src/languages.rs`. One macro for bindings and one for directories.

Reply:
@marco-c 

Thinking about this one in depth, this is just a temporary solution, because at the end only the Mozilla languages will be to need specified manually.

Reply:
If we decide to add bindings to the Mozilla languages too and use the `crate_name = { path ="..." }` notation, the structure will return the same as it is now.

Reply:
That sounds good to me